azext_iot/azext_metadata.json

```diff
@@ -1,4 +1,5 @@
 {
     "azext.minCliCoreVersion": "2.0.24",
-    "version": "0.7.0"
+    "azext.maxCliCoreVersion": "2.0.66",
```

Comment:
You don't need to add azext.maxCliCoreVersion": "2.0.66" if you removed min_profile.

Reply:
@limingu - so as long as minProfile is gone, we don't need to set a max version, and therefore won't need to release again when 2.0.67 is released?

Reply:
I think we have to change the azext.minCliCoreVersion": "2.0.67" once 2.0.67 is released, Correct me if I am wrong

Reply:
Did iot really need min_profile? If not, removing min_profile is the easiest way. One publish is good enough.

If iot needed min_profile, you need to add the max version.